tests/playwright/specs/user.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable mediawiki/valid-package-file-require */
+const { CreateAccountPage } = require( '../pageobjects' );
+const Util = require( 'wdio-mediawiki/Util' );
+
+describe( 'User', () => {
+	let username, password;
+
+	beforeEach( async () => {
+		username = Util.getTestString( 'User-' );
+		password = Util.getTestString();
+	} );
+
+	it( 'should be able to create account', async () => {
+		await CreateAccountPage.createAccount( username, password );
+		await page.screenshot( {
+			path: `${global.logPath}/userAccountCreation.png`
```

Comment:
This is a lot of hardcoding. Add this to the beforeEach and afterEach for every file. Checkout how we name files for Selenium